runtime/src/lib.rs

```diff
@@ -361,7 +361,7 @@ impl pallet_balances::Config for Runtime {
 	/// The ubiquitous event type.
 	type Event = Event;
 	type DustRemoval = ();
-	type ExistentialDeposit = ExistentialDeposit;
+	type ExistentialDeposit = ConstU128<EXISTENTIAL_DEPOSIT>;
```

Comment:
Is this really a necessary change?

Reply:
I tried some a handful of approaches but struggled to get it to compile. In the end I copied the approach from the template:
https://github.com/substrate-developer-hub/substrate-node-template/blob/6a8b2b12371395979099d2c79ccc1860531b0449/runtime/src/lib.rs#L249
Do you have another option in mind?